arcgis-ios-sdk-samples/Content Display Logic/Controllers/ContentCollectionViewController.swift

```diff
@@ -29,19 +29,29 @@ class ContentCollectionViewController: UICollectionViewController, UICollectionV
     // strong reference needed for iOS 10
     var searchController:UISearchController?
     
-    override func viewWillAppear(_ animated: Bool) {
-        super.viewWillAppear(animated)
-        // ensure that the search results appear beneath the navigation bar
-        definesPresentationContext = true
-    }
     override func viewWillDisappear(_ animated: Bool) {
         super.viewWillDisappear(animated)
-        // required in iOS 10 for the filter field to be interactable in the samples table
-        definesPresentationContext = false
+        guard #available(iOS 11.0, *) else {
```

Comment:
Wow. This is a fascinating use of `guard`. It most certainly is not an early return, but I understand why you did it this way. There's no `if !#available(…) { … }`, so using an `if` statement would mean an empty `if` clause. However, I still think an if statement is best. This really is not how `guard` is intended to be used. What would you think of this instead?
```swift
if #available(iOS 11.0, *) {} else {
    // required in iOS 10 for the filter field to be interactable in the samples table
    definesPresentationContext = false
}
```

Reply:
I suppose this `guard` statement is somewhat uncommon but I prefer it to an empty `if`. If you're set against it I'll change it. I do wish there was a cleaner way to positively specify old versions, an "earlier" rather than a "later" check.

Reply:
On second thought why don't I change it to `if` but leave a comment in the empty statement. That way it won't have to be empty!

Reply:
That would be fine as well. I've done something similar in the past.